README.md

```diff
@@ -4,14 +4,18 @@ Zotero utility code common across various codebases such as the Zotero client,
 Zotero translation architecture and others.
 
 Item utility functions require:
+
 - Calling `Zotero.Schema.init(json)` with the JSON from `schema.json` from Zotero schema repo
 - Calling `Zotero.Date.init(json)` with the JSON from `resource/dateFormats.json`
 - Loading `resource/zoteroTypeSchemaData.js` before `cachedTypes.js` or in Node.js running
+
   ```js
     let CachedTypes = require('./cachedTypes')
     CachedTypes.setTypeSchema(require('./resource/zoteroTypeSchemaData'))
   ```
+
 - Implementing `Zotero.localeCompare()`; a simple implementation would be
+
```

Comment:
Remove changes to empty lines.

Reply:
It was auto formatted by [markdownlint](https://github.com/DavidAnson/markdownlint) and I have removed those changes.